### ISSUE TITLE
Fine-tunning ArcFace 

### ISSUE THREAD

**Issue by EnzoBH** [peteryuX/arcface-tf2#34]
Hi!
Is there a way to perform a fine-tunning to your pretrained model? Or your training code don't support a quick way to do it?

For example, I've noticed in other repositories that it can be achieved with this type of commands:

CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train.py --network m1 --loss triplet --lr 0.005 --pretrained ./models/m1-softmax-emore,1

Thanks!
